excellent/functions/functions.go

```diff
@@ -35,7 +35,8 @@ var XFUNCTIONS = map[string]XFunction{
 	"text":     OneArgFunction(Text),
 	"boolean":  OneArgFunction(Boolean),
 	"number":   OneArgFunction(Number),
-	"datetime": OneTextFunction(DateTime),
+	"datetime": OneArgFunction(DateTime),
+	"time":     OneArgFunction(Time),
```

Comment:
i could definitely be persuaded that these conversion funcs should called `to_xxx` and then `datetime_from_parts(...)` and `time_from_parts(...)` become `datetime(...)` and `time(...)`

Reply:
I think the string versions are probably going to be used more so would rather keep those the short ones.